frontend/style/app/manage.css

```diff
@@ -14,27 +14,45 @@
 }
 
 .course {
-    /* For Jeremy: Change here the px */
+    /* just "box-shadow" it's not possible to convert to % */
     box-shadow: 5px 10px 18px grey;
-    border-width: 5px;
-    border-radius: 20px;
-    padding: 10px;
+    border-radius: 4%/8%;
+    padding: 2%;
     background-color: white;
+    border: 2px solid #00bfff;
 }
 
 .courseExerciseInputEdit {
     margin-left: 10%;
     text-align: center;
     font-size: 20px;
+    background-color: transparent;
 }
 
 .exercise {
-    /* For Jeremy: Here is the box for the exercise.
-    This box need to be different from the course box to make the ux better.
-    It can be by changing the background color or something else... */
+    /* just "box-shadow" it's not possible to convert to % */
     box-shadow: 5px 10px 18px grey;
-    border-width: 5px;
-    border-radius: 20px;
-    padding: 10px;
-    background-color: white;
+    border-radius: 5%/4%;
+    padding: 2%;
+    background-color: rgb(211, 230, 241);
+    border: 2px solid #00bfff;
 }
+
+
+select.swal2-input{
```

Comment:
Here you have to define another class with a new class name,  please don't override an existing class...